BUILDING.md

```diff
@@ -385,11 +385,12 @@ $ backtrace
 Prerequisites:
 
 * [Python 2.6 or 2.7](https://www.python.org/downloads/)
-* The "Desktop development with C++" workload from
-  [Visual Studio 2017](https://www.visualstudio.com/downloads/) or the
-  "Visual C++ build tools" workload from the
+* The "Build Tools for Visual Studio 2017" workload from the
   [Build Tools](https://www.visualstudio.com/downloads/#build-tools-for-visual-studio-2017),
-  with the default optional components.
+  with the default optional components. Once the "Build Tools for Visual Studio
+  2017" downloaded, install it by choosing "Visual C++ build tools" from the
+  installer and proceed. It will install the necessary build tools to compile
+  the C++ files.
```

Comment:
Maybe this?:

```markdown
After download, install it by choosing "Visual C++ build tools" from the
installer. It will install the necessary build tools to compile  the C++ files.
```

(Maybe even leave off that last sentence?)